activitysim/abm/models/trip_destination.py

```diff
@@ -924,6 +924,7 @@ def sample_skims(self, presample):
 
         o = self.model_settings["TRIP_ORIGIN"]
         d = self.model_settings["ALT_DEST_COL_NAME"]
+        n = self.model_settings.get("PRIMARY_ORIGIN", "origin")
```

Comment:
The structure of this change is simple enough, but our lack of a cohesive unit testing regime means there's no "test" that this works without also committing the model configs that it is supposed to go with.  We'll need to discuss whether we can make changes like this without proper testing.